src/related.tsx

```diff
@@ -111,28 +111,21 @@ class Related extends KalturaPlayer.core.BasePlugin {
 
   loadMedia() {
     const {ks, config, relatedManager} = this;
-    const {useContext, playlistId, entryList} = config;
+    const {useContext, playlistId, entryList, externalEntryList} = config;
```

Comment:
externalEntryList is source config with urls? so maybe sourcesList or mediaList external is not so clear

Reply:
in FEC-11570 it was originally called "External Assets"
it could be called external sources too, depends on what the app developers are more familiar with

Reply:
i'll ask Ron